requirements.txt

```diff
@@ -1,4 +1,5 @@
 Django>=2.2,<2.3
+psycopg2-binary>=2.8,<2.9
```

Comment:
To address new "Database not in UTC" bug. Reference: https://stackoverflow.com/a/68034786/7142170